.github/workflows/build.yml

```diff
@@ -0,0 +1,95 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Build aissemble
+
+on:
+  workflow_dispatch:
+    inputs:
+      buildBranch:
+        description: "Branch you want to build"
+        required: true
+        type: string
+        default: '2-build-action'
+  push:
+    branches: [ "2-build-action" ]
```

Comment:
S: Should update these to `dev`.  It might also make sense to have this run when a PR to dev is open and use the PR branch as the input build branch.  But that can come in later iterations once we get the dev branch building locally.

Reply:
Good catch

- [ ]  Switch to dev branch